regression/esbmc/unwindsetname_01_basic/test.desc

```diff
@@ -0,0 +1,4 @@
+THOROUGH
+main.c
+--unwindsetname compute_sum:0:11 --z3
```

Comment:
Any reason to specify z3?
Also thorough is only if the test is too long (>2s) or requires a specific solver.